change/@itwin-map-layers-ccc19b28-3b8c-4963-aa0f-a5d8a10efa3a.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
updating peers will technically be a major ver bump,
might be able to get away with minor here but that might piss off some,
if we do a major ver bump, id want to make sure we are going to the latest versions of 4.x and react to all deprecations

Reply:
I changed this to minor, could be keen on doing major, after running the linter I've reacted to the deprecations in appui

Reply:
id ant to update core and all other peer deps to the latest as well if we do a major bump

Reply:
we could split that into multiple prs, just need to hold off on publishing map-layers pkg until we are ready, or we do pre-releases

Reply:
I switched the changefile `type` to major and bumped all core pkgs and other peerDeps to the latest, minus appui-react and components react (which would require bumping up to their 5.x vers). Do we want to bump those 2? Should we also update the peerDeps to the latest as well?

Reply:
i would look at what that change means, im not sure how big of an update it will be for those pkgs

Reply:
it's quite big... some of the components used from `imodel-components-react'  are marked deprecated. On top of that, these 2 packages drop support for react 17, so we have to bump up to react 18, and respectively update peer deps of React. We'll have to use new components, edit the test cases and update type definitions for some of the react hooks in the package. Probably a larger amount of effort than what this PR is about

Reply:
there are only 3 imports from `imodel-components-react`
i imagine the colorswatch components are deprecated in favor of itwinui v3 components which we should be using

Reply:
also now that im looking i see there was a mixup on what latest meant, i meant the latest 4.x, not latest version of appui which is 5.0, my apologies, shouldve been more clear

Reply:
instead of going directly to a major, we should publish a pre-release to make sure everything works, and can also look updating other peer dep majors to include in the full major release